src/dags/importscripts/import_sport.py

```diff
@@ -0,0 +1,163 @@
+import csv
+from hashlib import blake2s
+import json
+import logging
+from typing import List, Tuple, TypedDict, Generator, Type
+
+
+# define logger for output to console
+logger = logging.getLogger(__name__)
+
+
+def get_dialect(file: str) -> Type:
+    """determine dialect of file
+
+    Args:
+        file: the path/to/file.ext that will be used to determine it's dialect
+
+    Returns:
+        A subclass dialect of Sniffer
+
+    """
+    with open(file) as data:
+        return csv.Sniffer().sniff(data.readline())
+
+
+def get_header(file: str) -> List:
+    """Return header row of file
+
+    Args:
+        file: the path/to/file.ext that will be used to determine it's header
+
+    Returns:
+        a list of columns in string
+
+    """
+    dialect = get_dialect(file)
+    with open(file, "r") as src:
+        data = csv.reader(src, dialect=dialect)
+        return next(data)
+
+
+def make_hash(composite_values: List, digest_size: int = None) -> int:
```

Comment:
`composite_values: List[str]`

And either: 
* `digest_size: int = 3` and remove the inline if statement in de function body (preferred) 
* or `digest_size: Optional[int] = None` and keep the inline if statement. (not recommend as you would need an explicit test for `is not None` to satisfy MyPy before passing `digest_size` to `blake2s`

Reply:
Aangepast, gekozen voor digest_size: int = 3